deps/leader_cron-0.1/src/leader_cron.app.src

```diff
@@ -0,0 +1,12 @@
+{application, leader_cron,
+ [
+  {description, ""},
+  {vsn, "0.2.1"},
+  {registered, []},
```

Comment:
add `{modules, []},` in there so that `make` can add `leader_cron`'s modules in the `.app` file please